tst/testspecial/backtrace.g.out

```diff
@@ -0,0 +1,167 @@
+gap> f := function() 
+>   local l;
+>   l := 0 * [1..6];
+>   l[[1..3]] := 1;
+> end;
+function(  ) ... end
+gap> f();
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
+Error, no 1st choice method found for `[]:=' on 3 arguments at GAPROOT/lib/methsel2.g:250 called from
+l[[ 1 .. 3 ]] := 1; at *stdin*:5 called from
+<function "f">( <arguments> )
+ called from read-eval loop at *stdin*:7
+type 'quit;' to quit to outer loop
+brk> Where();
+l[[ 1 .. 3 ]] := 1; at *stdin*:5 called from
+<function "f">( <arguments> )
+ called from read-eval loop at *errin*:1
+brk> quit;
+gap> 
+gap> 
+gap> f:=function() if true = 1/0 then return 1; fi; return 2; end;;
+gap> f();
+Error, Rational operations: <divisor> must not be zero in
+  1 / 0 at *stdin*:9 called from 
+<function "f">( <arguments> )
+ called from read-eval loop at *stdin*:10
+you can replace <divisor> via 'return <divisor>;'
+brk> Where();
+<function "f">( <arguments> )
+ called from read-eval loop at *errin*:1
+brk> quit;
+gap> 
+gap> 
+gap> f:=function() local x; if x then return 1; fi; return 2; end;;
+gap> f();
+Error, Variable: 'x' must have an assigned value in
+  if x then
+    return 1;
+fi; at *stdin*:12 called from 
+<function "f">( <arguments> )
+ called from read-eval loop at *stdin*:13
+you can 'return;' after assigning a value
+brk> Where();
+<function "f">( <arguments> )
+ called from read-eval loop at *errin*:1
+brk> quit;
+gap> 
+gap> 
+gap> f:=function() if 1 then return 1; fi; return 2; end;;
+gap> f();
+Error, <expr> must be 'true' or 'false' (not a integer) in
+  if 1 then
+    return 1;
+fi; at *stdin*:15 called from 
+<function "f">( <arguments> )
+ called from read-eval loop at *stdin*:16
+you can replace <expr> via 'return <expr>;'
+brk> Where();
+<function "f">( <arguments> )
+ called from read-eval loop at *errin*:1
+brk> quit;
+gap> 
+gap> 
+gap> f:=function() if 1 < 0 then return 1; elif 1 then return 2; fi; return 3; end;;
+gap> f();
+Error, <expr> must be 'true' or 'false' (not a integer) in
+  1 < 0 at *stdin*:18 called from 
```

Comment:
This is wrong, and right now I am not quite sure why it is...

But what *is* the right thing to print? In a previous comment, I suggested that for loops and if clauses, one could just hide the "bodies", and so e.g. print `while 1 do ... od;` instead of the full loop. But for `if/elif/fi` clauses with multiple conditions, this still doesn't seem quite good enough: ideally, we only want to print the `if` resp. `elif` with the expression that was executed. But there is no separate statement for that -- we are "in the middle" of a single statement here! I don't see a simple solution for that right now, so I opted to drop both approaches I tried from this PR right now, to be revisited in a future PR. For now, we are at least not doing worse than we were before.